notebooks/official/feature_store/sdk-feature-store.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
```

Comment:
<p>no future tense. drop the word 'will'.</p><p> </p><p>no 1st person, use 2nd person. Replace 'we' with 'you'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/486/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>